docs/pallas/grid_blockspec.md

```diff
@@ -104,11 +104,11 @@ the overall array). Each block index is then multiplied by the
 corresponding axis size from `block_shape`
 to get the actual element index on the corresponding array axis.
 
-```{note}
-This documentation applies to the case when the block shape divides
-the array shape.
-The documentation for the other cases is pending.
-```
+If the block shape does not divide evenly the overall shape then the
+last iteration on each axis will still receive references to blocks
+of `block_shape` but the elements that are out-of-bounds are padded
+on input and discarded on output. Note that at least one of the
+elements in each block must be within bounds.
```

Comment:
It would be good to mention that the padding value is unspecified

Reply:
Thanks, will fix in a future PR, this is being submitted